leaf-ui/js/onFunctions.js

```diff
@@ -85,12 +394,11 @@ function switchToAnalysisMode() {
 	removeHighlight();
```

Comment:
Comment starting on line 383 - this helper func for switching to Analysis mode, how does this interact with your  "switches to Analysis view iff there are no cycles and no syntax errors" (line 123-133) func? Would your little func trigger first? Could it possibly be combined?

Reply:
I didn't write these functions but basically the first function is tied directly to an html element event, and checks that all the basic and necessary model preconditions are met in order to move into analysis view. If that is the case it calls two helper functions. This is just a helper function for clearing and showing html elements based on the view. I supposed they could be combined but I think the helper function does make the code cleaner for the first function, especially since the second function is almost entirely just hiding and showing html elements. Thats probably a stylistic question though so maybe we should ask Alicia?